docs/6.-Engineering/Quality-Standards/Coding-Conventions.md

```diff
@@ -23,4 +23,46 @@ Unit and Integration tests should make use [Shouldly](https://github.com/shouldl
 
 # TypeScript and JavaScript Coding Conventions
 
-> _TODO: document this section and link to an industry-standard style guide_
```

Comment:
Remember the objective is:

"Drive the Typescript coding standard set up a meeting and get alignment with the rest of the team on which standards we'll use."

What you've written is more as part of a conversation starter than standards that have been set out since from what I'm aware we don't have alignment yet? 

Note when writing Standards we use specific language. 

*Should* when adding suggestions and 
Must for things that is required.

e.g. 

Instead of "When using eslint in a project, we can install specific packages that will help set the rules"

We can be more specific with this: 
"When starting a new JS project we must AirBnb eslint config to specify standards" 

But of course this specificity need to be aligned with the rest of the team first.